### PR TITLE
Fix missing cursor for ternary plot

### LINKED ISSUE #7056: Cursor missing for ternary plot
Hello,

I found that the cursors are missing for `Zoom` and `Pan`, see the GIF below for the current cursor behavior (Plotly v2.34.0) and my expected bahavior (PR #7057) for the [Ternary Plot](https://plotly.com/javascript/ternary-plots/).

Note the difference in the cursor.

Current cursor behavior(Plotly v2.34.0) :
recording

Expected bahavior (PR #7057):
recording

### PR DESCRIPTION
**Pull request by Lexachoc**
Fix #7056

This is my first contribution to such a large open source project. If I understand the coding style wrong or wrong logic for the fix, please let me know.

### CODE REVIEW

File: src/plots/ternary/ternary.js [hunk 1]
```diff
@@ -97,6 +98,16 @@ proto.makeFramework = function(fullLayout) {
...
 };
 
+proto.updateFx = function(dragmode) {
+    var toplevel = this.plotContainer.selectAll('g.toplevel');
+
+    if(dragmode === 'pan') {
+        toplevel.style('cursor', 'move');
+    } else {
+        toplevel.style('cursor', 'crosshair');
+    }
```
Comment (archmoj) [inline src/plots/ternary/ternary.js:108]:
```suggestion
    toplevel.style('cursor', dragmode === 'pan' ? 'move' : 'crosshair'); 
```

Comment (Lexachoc) [reply src/plots/ternary/ternary.js:108]: That's much cleaner! Thanks

File: draftlogs/7057_fix.md [hunk 2]
```diff
@@ -0,0 +1 @@
+ - Fix missing cursor for Ternary Plot[[#7057](https://github.com/plotly/plotly.js/pull/7057)]
```
Comment (archmoj) [inline draftlogs/7057_fix.md:1]:
```suggestion
 - Fix missing cursor for Ternary Plot[[#7057](https://github.com/plotly/plotly.js/pull/7057)],
   with thanks to @Lexachoc for the contribution!
```

**Comment by Lexachoc** [comment]
I also updated the `src/plots/ternary/index.js`:
```diff
-    if(dragmode === 'pan') {
-        toplevel.style('cursor', 'move');
-    } else {
-        toplevel.style('cursor', 'crosshair');
-    }
+ toplevel.style('cursor', dragmode === 'pan' ? 'move' : 'crosshair');
```

File: src/plots/ternary/ternary.js [hunk 3]
```diff
@@ -97,6 +98,12 @@ proto.makeFramework = function(fullLayout) {
     Drawing.setClipUrl(_this.layers.grids, clipId, gd);
 };
 
+proto.updateFx = function(dragmode) {
+    var toplevel = this.plotContainer.selectAll('g.toplevel');
+
+    toplevel.style('cursor', dragmode === 'pan' ? 'move' : 'crosshair');
```
Comment (archmoj) [inline src/plots/ternary/ternary.js:104]: Let's use `_ternarylayer`.
```suggestion
    var fullLayout = gd._fullLayout;
    fullLayout._ternarylayer
        .selectAll('g.toplevel')
        .style('cursor', fullLayout.dragmode === 'pan' ? 'move' : 'crosshair');
```

File: src/plots/ternary/index.js [hunk 4]
```diff
@@ -75,3 +75,11 @@ exports.clean = function(newFullData, newFullLayout, oldFullData, oldFullLayout)
         }
     }
 };
+
+exports.updateFx = function(gd) {
+    var fullLayout = gd._fullLayout;
+
```
Comment (archmoj) [inline src/plots/ternary/index.js:81]: Please run `npm run lint -- --fix` and commit the changes.

```suggestion
```

Comment (Lexachoc) [reply src/plots/ternary/index.js]: image

I hope I'm doing it correctly.

**Review by archmoj — APPROVED** [review]
Thanks for the PR.
:dancer:

**Pull request merged**